src/syntax/rules.rkt

```diff
@@ -4,9 +4,12 @@
 
 (require "../common.rkt" "syntax.rkt" "../float.rkt" "../type-check.rkt")
 
-(provide (struct-out rule) *rules* *simplify-rules* *fp-safe-simplify-rules*)
+(provide (struct-out rule) *rules* *simplify-rules* *fp-safe-simplify-rules* fpconstants)
 (module+ internals (provide define-ruleset *rulesets*))
 
+(define fpconstants (set 'E 'LOG2E 'LOG10E 'LN2 'LN10 'PI 'PI_2 'PI_4 '1_PI '2_PI '2_SQRTPI 'SQRT2 'SQRT1_2 'INFINITY 'NAN 'TRUE 'FALSE))
```

Comment:
Wait, what's wrong with the `constant?` in `syntax/syntax.rkt`? Even if you need the full list, why not use `constant` in `(submit "syntax/syntax.rkt" internals)`?

Reply:
Checked for all uses, and it looks like `constant?` is all you need.

Reply:
Oh, great!